pkg_building.Rmd

```diff
@@ -57,7 +57,7 @@ The [`tabulizer` package](https://github.com/ropensci/tabulizer) e.g. has an int
 
 ## Code Style
 
-* For more information on how to style your code, name functions, and R scripts inside the `R/` folder, we recommend reading the [code chapter in Hadley's book](http://r-pkgs.had.co.nz/r.html). We recommend the [`styler` package](https://github.com/r-lib/styler) for automating part of the code styling.
+* For more information on how to style your code, name functions, and R scripts inside the `R/` folder, we recommend reading the [code chapter in Hadley's book](http://r-pkgs.had.co.nz/r.html). We recommend the [`styler` package](https://github.com/r-lib/styler) for automating part of the code styling. You can choose to use `=` over `<-` as long you are consistent with one choice within your package. 
```

Comment:
Maybe we want to make a separate bullet for the `<-` vs. `=` thing since it's a big issue. In addition, I think we should mention the caveats, e.g. in R6 you have to assign functions with `=` and are not allowed to use `<-`

Reply:
Can you write the caveat part? :pray: thanks!

Reply:
will do